engine/src/test/java/org/pentaho/di/core/row/ValueDataUtilTest.java

```diff
@@ -198,6 +201,107 @@ public void testPlus() throws KettleValueException {
 
   }
 
+  @Test
+  public void checksumTest() {
+    try {
+      String testingFilename = "testingFile";
+      File tempFile = File.createTempFile( testingFilename, ".txt" );
+      tempFile.deleteOnExit();
+      FileUtils.writeStringToFile( tempFile, "test" );
+
+      String checksum = ValueDataUtil.createChecksum( new ValueMetaString(), tempFile.getCanonicalPath(), "MD5", true );
+      assertEquals( "098f6bcd4621d373cade4e832627b4f6", checksum );
+    } catch ( Exception e ) {
+      fail( e.getMessage() );
```

Comment:
in these situations it's better to just have the test throwing the exception; as is we'll lose the stacktrace if it fails